handlers_test.go

```diff
@@ -14,25 +14,25 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/go-kit/kit/log"
```

Comment:
We usually sort external imports from stdlib imports by putting them at the end with a newline between.

Reply:
Looks like you picked this one up in #69. I will merge #68 first once it gets the thumbs up.